src/execution_plan/execution_plan_build/build_pattern_comprehension_ops.c

```diff
@@ -105,6 +105,17 @@ void buildPatternComprehensionOps
 			// build filter tree
 			FT_FilterNode *filter_tree = NULL;
 			AST_ConvertFilters(&filter_tree, predicate);
+
+			if(!FilterTree_Valid(filter_tree)) {
```

Comment:
add test for filter tree is invalid in this case

Reply:
@AviAvni I can't test it cause it fails only in case of a bug in the code (the same way you can't test an assertion)